dnsimple/struct/certificate.py

```diff
@@ -60,8 +61,11 @@ def __init__(self, data):
 
 
 class LetsencryptCertificateInput(dict):
-    def __init__(self, contact_id, auto_renew=None, name=None, alternate_names=None):
-        dict.__init__(self, contact_id=contact_id, auto_renew=auto_renew, name=name, alternate_names=alternate_names)
+    def __init__(self, contact_id=None, auto_renew=None, name=None, alternate_names=None):
+        dict.__init__(self, auto_renew=auto_renew, name=name, alternate_names=alternate_names)
+        if contact_id is not None:
+            warnings.warn("DEPRECATION WARNING: LetsencryptCertificateInput#contact_id is deprecated and will be ignored.")
```

Comment:
What do you think about:

```suggestion
            warnings.warn("DEPRECATION WARNING: LetsencryptCertificateInput#contact_id is deprecated and its value is ignored and will be removed in the next major version.")
```

Reply:
👍